### DISCUSSION TITLE
Pixels and ports

### DISCUSSION THREAD

**Discussion by JonathanStanders95** [forkineye/ESPixelStick#455] [General]
Hello there, i would like to know where I can change the number of LED's supported on esp and esp32 and ports!

**Comment by brmeadows2841** [comment]
Ports are set from Device Setup page.
The number of pixels is set as pixel count after the output is enable
![test](https://user-images.githubusercontent.com/81775646/149348346-67df42d6-03f0-41c4-9dac-94d19aa1951a.png)
